main/tracker/template.py

```diff
@@ -38,7 +38,7 @@
 license = "GPL-2.0-or-later AND LGPL-2.1-or-later"
 url = "https://gnome.pages.gitlab.gnome.org/tracker"
 source = f"$(GNOME_SITE)/{pkgname}/{pkgver[:-2]}/{pkgname}-{pkgver}.tar.xz"
-sha256 = "52592cfe19baffd16dbe47475be7da750dbd0b6333fd7acb60faa9da5bc40df2"
+sha256 = "46b8c16ff289d9507baa112c717845a200fe3d1ffcc2324562e02c40db9a151d"
 # lto fails: Invalid GType function: 'tracker_endpoint_http_get_type'
 options = ["!cross", "!lto"]
```

Comment:
Not needed anymore, see https://github.com/chimera-linux/cports/pull/1647/files#r1536723660:
```suggestion
options = ["!cross"]
```